application/dashboard/data_helpers.py

```diff
@@ -60,7 +60,8 @@ def get_published_dashboard_data():
     }
 
     weeks = []
-    cumulative_total = []
+    cumulative_number_of_pages = []
+    cumulative_number_of_major_updates = []
```

Comment:
These might feel better named as "page_counts_over_time" to infer that there are multiple data points - "cumulative_number_of_pages" just feels like a single number to me.

Reply:
How about `cumulative_number_of_pages_by_week` and `cumulative_number_of_major_updates_by_week`?

Reply:
Happy for you to make the call (but I think I prefer my original suggestion - `by` tends to imply some kind of mapping relationship when I've used it previously)

Reply:
@samuelhwilliams I think it _might_ be useful to explicitly state the time periods involved (weeks), as there's no way you can infer that from an array of numbers?